x-pack/test/security_solution_endpoint/apps/endpoint/policy_details.ts

```diff
@@ -21,8 +21,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
   const policyTestResources = getService('policyTestResources');
 
-  // Failing: See https://github.com/elastic/kibana/issues/100236
-  describe.skip('When on the Endpoint Policy Details Page', function () {
+  describe('When on the Endpoint Policy Details Page', function () {
```

Comment:
From our conversation - we probably should be waiting for the endpoint form to load first on the fleet page. In the `beforeEach` under that `describe()` block (line 632) we probably should add:

```typescript
await testSubjects.existOrFail('endpointIntegrationPolicyForm')
```

Reply:
Added in your suggestion, everything still passes locally